src/public/indoorMapEntities/indoor_map_entity.js

```diff
@@ -1,20 +1,25 @@
-var IndoorMapEntity = function(indoorMapEntityId, indoorMapFloorId, position) {
-    var _indoorMapEntityId = indoorMapEntityId;
-    var _indoorMapFloorId = indoorMapFloorId;
-    var _position = position;
-
-    this.getIndoorMapEntityId = function() {
-        return _indoorMapEntityId;
-    };
-
-    this.getIndoorMapFloorId = function() {
-        return _indoorMapFloorId;
-    };
-
-    this.getPosition = function() {
-        return _position;
-    };
-
-};
-
+var IndoorMapEntity = function(indoorMapEntityId, indoorMapFloorId, position, outline) {
+    var _indoorMapEntityId = indoorMapEntityId;
+    var _indoorMapFloorId = indoorMapFloorId;
+    var _position = position;
+    var _outline = outline;
+
+    this.getIndoorMapEntityId = function() {
+        return _indoorMapEntityId;
+    };
+
+    this.getIndoorMapFloorId = function() {
+        return _indoorMapFloorId;
+    };
+
+    this.getPosition = function() {
+        return _position;
+    };
+
+    this.getOutline = function(){

```

Comment:
what is this actually returning? I can't spot any model type for "outline polygon" or similar, an no docs, so not sure how this data is arranged - is it a nested arrays of lat-lng coords? here. Naming - maybe "getOutlinePolygonPoints" if it is a collection of points?
Similar in heatmap.js is:
https://github.com/wrld3d/wrld.js/blob/master/src/public/heatmap.js#L253
https://github.com/wrld3d/wrld.js/blob/master/src/public/heatmap.js#L107